tests/cli/ui/test_shared.py

```
@@ -0,0 +1,24 @@
+"""Tests for shared UI helpers."""
+
+from src.cli.ui.shared import create_bottom_toolbar
+
+
+def test_create_bottom_toolbar_escapes_working_dir(mock_context, monkeypatch):
+    """Ensure working directory is escaped before embedding in HTML."""
+    monkeypatch.setattr("src.cli.ui.shared.get_version", lambda: "0.0.0")
+
+    working_dir = "/tmp/<foo&bar>"
```

Ruff (0.14.6)

Probable insecure usage of temporary file or directory: "/tmp/<foo&bar>"

(S108)